ansible/roles/software/ckan/inventory/templates/beaker-cache-cleanup.sh.j2

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# https://github.com/GSA/datagov-deploy/issues/682
+# delete stale session rows in beaker_cache table.
+
+age_in_days=7
+LOCKFILE=/tmp/beaker_cache_cleanup.sh.pid
+
+if [ -e ${LOCKFILE} ] && kill -0 `cat ${LOCKFILE}`; then
```

Comment:
[flock](https://linux.die.net/man/1/flock) simplifies this logic a little bit. If you use flock, you can just wrap your critical commands in a subshell and you don't need to worry about deleting the lockfile as part of cleanup. flock will automatically remove the lock when the commands complete.